### ISSUE TITLE
VSCodeInsider_VS-DeleteEnvironment_ScreenReader:VoiceOver does not announces the information about the pop up  as soon as pop up activated.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#89089]
**Environment Details:​**
VSCode Version: 1.41.0-insider (System setup)
Operating System: MacOS
Screen reader: Voice Over
Mac version: 10.15.1 (19B88)

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till any environment and press right click.
5. Navigate till "Delete Environment " button and click on it  .
6. Start the Voice Over.
7. Observe as soon as the pop up gets activated the information which is present on the pop up is announces by the VoiceOver or not.

**Actual:**
VoiceOver does not announces the information which is present on the pop up  as soon as pop up gets activated.

**Expected:​**
VoiceOver should announces the information which is present on the pop up  as soon as pop up gets activated.

**Same issue observed for the below scenario:**
**1.VS-Suspend Environment**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till any environment and the "Suspend " button and click on it  .
5.Verify that all the elements of "VS-Suspend Environment" are accessible and MAS Compliant.

**2.Show pricing info**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5.Select any existing environment and then again click on the "+" button and navigate till "Show Pricing Info" ,click on it.
6.Verify that all the elements of "Show pricing info" pop up are accessible and MAS Compliant.

**​User Impact:​**
Screen reader user does not get the information about the control if it screen reader does not announces.

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/55585
